projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-list-post-type-taxonomies-endpoint.php

```diff
@@ -41,26 +64,26 @@ function callback( $path = '', $blog_id = 0, $post_type = 'post' ) {
 
 		$this->localize_initial_taxonomies( $post_type );
 
-		$args = $this->query_args();
+		$args = $this->query_args(); // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
```

Comment:
If this isn't used, can we get rid of it? If not, maybe you could add a comment as to why?

Reply:
dc5fb9e055537b69460d5318b4c5e7b919da778c